examples/interop/compose-in-swiftui/.gitignore

```diff
@@ -0,0 +1,5 @@
+local.properties
```

Comment:
Let's add "ios-" as a prefix for all examples (ios-compose-in-swiftui, ...). We can add desktop, web in the future.

Reply:
Done